internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/shares.go

```diff
@@ -1162,14 +1162,14 @@ func (h *Handler) listPublicShares(r *http.Request, filters []*link.ListPublicSh
 			}
 
 			statResponse, err := c.Stat(ctx, statRequest)
-			if err != nil {
```

Comment:
we should keep detecting grpc call errors and returning `nil, err`

Reply:
agree 👍